### DISCUSSION TITLE
Make every value in the fields value of a MessageEmbed a type of Array while still keeping EmbedFieldData.

### DISCUSSION THREAD

**Discussion by DanieliusDev** [discordjs/discord.js#7509] [Ideas]
Make every value in the fields value of a MessageEmbed a type of Array while still keeping EmbedFieldData. 

for example you could use:
```js
new MessageEmbed({
    fields: [
        // name  value  inline
        ['Name', 'Value', true]
    ]
})
```

while still being to use this of course:
```js
new MessageEmbed({
    fields: [
        { name: 'Name', value: 'Value', inline: true }
    ]
})
```

**Comment by BenjammingKirby** [comment]
okay.. why? 
why go with tuples? 
some of the times there may be multiple optional properties such as embed's authors
in those cases i'd rather use an object and not a tuple 
and to stay consistent, i'd make them all use objects instead of tuples
